src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -221,14 +232,19 @@ public void MaximumAutomaticRedirections_SetThenGetOne_Success(Uri remoteServer)
             Assert.Equal(MaximumAutomaticRedirections, request.MaximumAutomaticRedirections);
         }
 
-        [Theory, MemberData(nameof(EchoServers))]
-        public void ContinueTimeout_SetAfterRequestSubmitted_ThrowsInvalidOperationException(Uri remoteServer)
+        [Fact]
+        public async Task ContinueTimeout_SetAfterRequestSubmitted_ThrowsInvalidOperationException()
         {
-            HttpWebRequest request = WebRequest.CreateHttp(remoteServer);
-            using (WebResponse response = request.GetResponse())
+            await LoopbackServer.CreateServerAsync(async (server, uri) =>
             {
-                Assert.Throws<InvalidOperationException>(() => request.ContinueTimeout = 255);
-            }
+                HttpWebRequest request = WebRequest.CreateHttp(uri);
+                Task<WebResponse> getResponse = request.GetResponseAsync();
+                await LoopbackServer.ReadRequestAndSendResponseAsync(server);
+                using (WebResponse response = await getResponse)
+                {
+                    Assert.Throws<InvalidOperationException>(() => request.ContinueTimeout = 255);
+                }
+            });
```

Comment:
Same question.

Also, all of these follow the same pattern.  They could all either:
a) just have a bunch of Assert.Throws in the same test to avoid making lots of additional requests, or
b) use a helper that contains the boilerplate